src/data-pipeline/etl-common/src/main/resources/ts/traffic_source_category_rule_v0.json

```diff
@@ -0,0 +1,7642 @@
+{
+    "www.gnadenmeer.de": {
```

Comment:
Try not to include special characters in the key. Could we changed to this structure?
```
{
  domain: string;
  source: string;
  category: string;
  params:[];
}
```

Reply:
changed to the structure to array.  keep the field 'url', it not just domain, path might be in the url.  

e.g.
 ```
 {
        "url": "google.com/products",
        "source": "Google Shopping",
        "category": "Search",
        "params": [
            "q"
        ]
    }
```